neo/d3xp/Game_local.cpp

```diff
@@ -334,6 +334,7 @@ void idGameLocal::Init( void ) {
 	//Update MSEC and gameFps
 	gameFps = cvarSystem->GetCVarInteger("com_gameHz");
 	msec = 1000.0f / cvarSystem->GetCVarFloat("com_gameHz");
+	msec *= 0.96f*0.96f; //HACK to emulate OG D3 msec error, in order to have exactly the same game logic speed
```

Comment:
why `0.96f*0.96f` ?
I get that `16 / 16.666 == 0.96`, but why square it?

Reply:
That was a mistake from me. I thought that I needed to do that twice because common frame was already working at 16.666 hz, but multiplying by 0.96f twice was a mistake.

In fact that product for 0.96f is something I don't like at all. It is some kind of "duct-tape" ugly code. I will look at this feature in some days or weeks in order to see if with a fresh mind I can actually figure out something better.